wp-cli/vip-migrations.php

```diff
@@ -20,7 +20,15 @@ class VIP_Go_Migrations_Command extends WPCOM_VIP_CLI_Command {
 	 */
 	function dbdelta( $args, $assoc_args ) {
 		global $wpdb;
-
+		
```

Comment:
Minor: can we remove the unnecessary indents from the empty lines added in this changeset?